Mathlib/Data/NNReal/Defs.lean

```diff
@@ -872,8 +872,6 @@ end Set
 namespace Real
 
 /-- The absolute value on `ℝ` as a map to `ℝ≥0`. -/
--- Porting note (kmill): `pp_nodot` has no affect here
--- unless RFC https://github.com/leanprover/lean4/issues/6178 leads to dot notation pp for CoeFun
```

Comment:
We can delete this porting note because in the current situation, and after the `CoeFun` issue is solved, we don't get a dot in the pretty-printer.